docs/mkcloud.md

```diff
@@ -208,7 +228,7 @@ fi
 
 export cloudpv=${loused}
 export cloudsource=develcloud7
-export cloud=$cloudsource
+export cloud="mycloud"
```

Comment:
Unfortunately I can't reply directly to your last comment due to https://github.com/isaacs/github/issues/998, but at least I can copy'n'paste it in here from the email notification:

> FYI - the mkcloud code is actually looking at $cloud plus + br, so that is why I truncated it to 9 characters. (See line 133 in mkcloud.)

Ah OK, that makes sense; I missed that detail.

> Having said that, I think just setting it to a static value is the best call here.

Honestly I'm more than a bit worried that this will easily lead to one developer accidentally nuking another developer's cloud on a shared physical host, because neither of them realised that they were supposed to set `$cloud` to a unique value.  So I stand by my previous suggestion of not setting this at all, and ensuring that a check catches it not being set and issues a very helpful error message explaining why and how to set it.  To my mind this still counts as "working out of the box" because it doesn't require the developer to spend any extra effort reading documentation in order to get up and running.

> Also, bear in mind that this is example code for a suggested mkcloud wrapper script. These changes are intended to give new developers something that will actually work for them out of the box. I am not sure what the rationale is for not including some of this stuff within mkcloud itself but this seems to be the status quo.

Don't get me wrong, I'm all for sane defaults and making stuff work out of the box as much as possible, just not to the extent of risking severe data loss.  `mkcloud`'s `cleanup` step is totally merciless in the way it nukes virtual resources which it believes belong to the cloud it's currently working on.

Reply:
Right, I guess the point I'm making is more to the effect that if it's really important to have the user select a unique cloud name (and I think we agree that it is), why not have a mechanism such as you propose in mkcloud itself rather than in an example script?

Reply:
> why not have a mechanism such as you propose in mkcloud itself rather than in an example script?

If you're referring to the check and the helpful error message, that's exactly what I was proposing :-)  Sorry if that wasn't obvious - it didn't occur to me that it might sound like I was proposing putting these in the example script, which would not make nearly as much sense.

Reply:
I think what the value of this setting is (that maybe should be listed as commented out by default) is to explain that you can set up more than one cloud on the same host. maybe just set it to ```${cloudsource/cloud/}``` ?